cms/static/sass/views/_group-configuration.scss

```diff
@@ -59,12 +59,12 @@
           @extend %cont-truncated;
           @extend %t-title5;
           @extend %t-strong;
-          margin-right: ($baseline*14);
+          @include margin-right($baseline*14);
           color: $black;
 
           .toggle {
             display: inline-block;
-            padding-left: $baseline;
+            @include padding-right($baseline);
```

Comment:
@halawa You mean this one, right?

This was intentional because I didn't like how it looks and found it odd. I found that the padding right aligns better.